Parser/pegen.h

```diff
@@ -263,11 +263,21 @@ int _PyPegen_check_barry_as_flufl(Parser *);
 mod_ty _PyPegen_make_module(Parser *, asdl_seq *);
 
 // Error reporting helpers
-expr_ty _PyPegen_get_invalid_target(expr_ty e);
+typedef enum {
+    STAR_TARGETS,
+    DEL_TARGETS,
+    FOR_TARGETS
+} TARGETS_TYPE;
+expr_ty _PyPegen_get_invalid_target(expr_ty e, TARGETS_TYPE targets_type);
+#define GET_INVALID_TARGET(e) _PyPegen_get_invalid_target(e, STAR_TARGETS)
```

Comment:
Maybe we should surround `_PyPegen_get_invalid_target` in a `CHECK` to make sure this does not return NULL